src/main/java/de/retest/recheck/report/SuiteReplayResult.java

```diff
@@ -89,13 +98,21 @@ public String getExecSuiteSutVersion() {
 		return execSuiteSutVersion.getSutVersion();
 	}
 
+	public GroundState getGroundState() {
```

Comment:
I would rather user the field name for the getter, i.e., `getReplaySutVersion()` and rename the below to something like `getReplaySutVersionString()`.